### ISSUE TITLE
Borderless Windows script

### ISSUE THREAD

**Issue by Sasameku** [cursey/kanan#107]
When using Kanan Auto Start, Borderless Windows sometimes applies incorrectly, so half of the client isn't on the screen. In addition, when using Kanan Auto Start to mutliclient with Kanan, sometimes Borderless Windows fails to apply, and only starting a new client with Kanan Auto Start would get the Borderless Windows onto the second client, but not the third one.

**Comment by cursey** [comment]
Yeah so, Borderless Windowed mode is going to try and make whatever window the win32 `FindWindow` API returns, which is pretty much random and not guaranteed to be most recently created. This is why when multi-clienting sometimes it will work on the new clients, sometimes it won't. 

I know how to fix it. I need to find where mabi is storing the handle to its window instead of using the `FindWindow` API. 

As for it not applying correctly sometimes, I have gotten that issue as well and am still trying to figure out why. It's difficult because it works like > 80% of the time.

**Comment by inklit** [comment]
I've had this issue....it's pretty odd haha.
However I generally just stop the client and restart it.

**Comment by Kyralis** [comment]
This may not be the same issue but there is an issue when using multiple screens if you are initially in windowed mode and try to use a different screen than your main one or have more than one desktop. I experience this only with my left most monitor. When it jumps to the other screen on start it will initialize the window frame to a smaller window, usually thin and tall. It then locks it in place when you use the borderless windowed mode as a full screen application that is only able to cover that same section of screen until you exit full screen. However this was a mabi issue. Change the main monitor display for the game to a different monitor. As for multiclient issue if you don't have multiple monitors I have no idea.
